### DISCUSSION TITLE
"Update" old session

### DISCUSSION THREAD

**Discussion by VilleSuominen** [sienori/Tab-Session-Manager#825] [Q&A]
Link to earlier related post:
https://github.com/sienori/Tab-Session-Manager/issues/798

If I open a saved session, is there a way to make autosave, save on close and save on shutdown to update the session it was opened from. Instead of creating sessions named "autosave - whateverpageIwason"

1# I would of course like to know if this functionality already exists and I just don't know how to use it.

2# Is there a way to get this kind of functionality from somewhere else

3# Can the devs include such a feature in the future?

**Comment by sienori** [comment]
@DiegoBM and I are currently working on "Active session" feature. #890
